packages/babel-cli/src/babel/util.js

```diff
@@ -5,6 +5,11 @@ import * as babel from "@babel/core";
 import path from "path";
 import fs from "fs";
 
+// Technically we could use the "semver" package here, but (for exmaple)
+// parseFloat("4.23.6") returns 4.23 so it's "good enough"
+export const BABEL_SUPPORTS_EXTENSIONS_OPTION =
+  parseFloat(babel.version) >= 7.11;
```

Comment:
# :warning: 

- Before merging this PR, I need to add a warning in `new-version` in `Makefile` to replace 7.11 with the actual new version.